model/address.py

```diff
@@ -837,15 +844,17 @@ def export_rdf(self, view='gnaf'):
 
             # RDF: data properties
             if self.description is not None:
-                g.add((a, DCT.description, Literal(self.description, datatype=XSD.string)))
+                g.add((a, DCTERMS.description, Literal(self.description, datatype=XSD.string)))
 
             g.add((a, GNAF.hasPostcode, Literal(self.postcode, datatype=XSD.integer)))
 
             if self.building_name is not None:
                 g.add((a, GNAF.hasBuildingName, Literal(self.building_name, datatype=XSD.string)))
 
             g.add((a, GNAF.hasDateCreated, Literal(self.date_created, datatype=XSD.date)))
```

Comment:
Repeating `date_created` and `modified` values with 2 different predicates (one in GNAF NS and one in DCTERMS NS). Is that intentional for backwards compatibility of sorts?

Reply:
Yes, intentional for backwards compatibility.
At the time I wrote this code last week, Simon had not yet discussed with Jo making the changes to the GNAF ontology, so the ontology still expects/expected the use of `GNAF.hasDateCreated`.

Reply:
Ok sounds like a good way forward